openmetadata-service/src/main/java/org/openmetadata/service/Entity.java

```diff
@@ -344,7 +344,7 @@ public static <T> T getEntityByName(String entityType, String fqn, String fields
   public static EntityRepository<? extends EntityInterface> getEntityRepository(@NonNull String entityType) {
     EntityRepository<? extends EntityInterface> entityRepository = ENTITY_REPOSITORY_MAP.get(entityType);
     if (entityRepository == null) {
-      throw EntityNotFoundException.byMessage(CatalogExceptionMessage.entityTypeNotFound(entityType));
+      throw EntityNotFoundException.byMessage(CatalogExceptionMessage.entityRepositoryNotFound(entityType));
```

Comment:
created a new exception to be thrown here otherwise is super misleading. Spent a bunch of time debugging broken relationships when the issue was a not initialized `ENTITY_REPOSITORY_MAP`